doc/classes/Image.xml

```diff
@@ -220,6 +220,22 @@
 				Returns the image's format. See [enum Format] constants.
 			</description>
 		</method>
+		<method name="get_format_with_alpha" qualifiers="static">
+			<return type="int" enum="Image.Format" />
+			<param index="0" name="format" type="int" enum="Image.Format" />
+			<description>
+				Returns the equivalent image format to the [param format] parameter specified, but with an alpha channel present. For example, [method get_format_without_alpha] with [constant Image.FORMAT_RGB8] returns [constant Image.FORMAT_RGBA8]. If no equivalent exists, returns the format that was originally passed.
+				[b]Note:[/b] This method always returns the original format for VRAM-compressed formats.
+			</description>
+		</method>
+		<method name="get_format_without_alpha" qualifiers="static">
+			<return type="int" enum="Image.Format" />
+			<param index="0" name="format" type="int" enum="Image.Format" />
+			<description>
+				Returns the equivalent image format to the [param format] parameter specified, but without an alpha channel present. For example, [method get_format_without_alpha] with [constant Image.FORMAT_RGBA8] returns [constant Image.FORMAT_RGB8]. If no equivalent exists or its equivalent would be larger in memory than the specified format, returns the format that was originally passed.
+				[b]Note:[/b] This method always returns the original format for VRAM-compressed formats.
```

Comment:
Provide a few examples with "such as _ or _"

Reply:
I've expanded upon the `is_compressed()` documentation while I was at it.